test/test_cmd_log.py

```diff
@@ -43,15 +43,31 @@ def test_colint(self):
         parsed = check_create_parser("--c", "colint_test")
         self.assertEqual(parsed.colint, "colint_test")
 
+    def test_numrowcount(self):
+        """ Function to test the number of row count in the command line
+                correct and incorrect """
+        with self.assertRaises(SystemExit):
+            check_create_parser("-n", '1')
+        parsed = check_create_parser("--n", '1')
+        self.assertEqual(parsed.numrowcount, 1)
+
+    def test_numrowcount_default(self):
```

Comment:
Great to see values being defaulted if none specified, and being tested via unit-tests!